odml/property.py

```diff
@@ -150,7 +150,35 @@ def _validate_parent(new_parent):
 
     @property
     def value(self):
-        return self._value
+        """
+        Returns the value(s) stored in this property. Method always returns a list that
+        is a copy (!) of the stored value. Changing this list will NOT change the property.
+        For manipulation of the stored values use the append, extend, and direct access methods
+        (using brackets).
+
+        For example:
+        >> p = odml.Property("prop", value=[1, 2, 3])
+        >> print(p.value)
+        [1, 2, 3]
+        >> p.value.append(4)
+        >> print(p.value)
+        [1, 2, 3]
```

Comment:
I think this line should be [1, 2, 3, 4] ;)

Reply:
hehe, no, I think this is correct, ``p.append(4)`` would change it, ``p.value.append(4)`` appends to the copy and not the property

Reply:
argh sorry, blanked out the `.value.` part...